charts/kubevirt/0.2.0/values.yaml

```diff
@@ -0,0 +1,7 @@
+kubevirt:
+  image: registry.suse.com/suse/sles/15.5/virt-operator:1.1.0-150500.8.6.1
+  version: 1.1.0-150500.8.6.1
```

Comment:
If the `1.1.0-150500.8.6.1` is the same for both the `image` and `version`, does it make sense in the K8s resources to refer to the `kubevirt.image` as `{{ .Values.kubevirt.image }}:{{ .Values.kubevirt.version }}`? That way we will not have the same version in two places. WDYT?

Reply:
I was wondering about this. We didn't have the version value in place before so I'm not entirely sure where it is actually used. I guess we can do this given that we're most certainly not going to upgrade again before Edge 3.0 so even if those two are different in a newer version we should be in a good place to address it.

Reply:
The generated manifests have everything inlined so what I do in this initial step is to grab the most commonly templated values out of the manifests.

I do believe that the suggestion puts us in a better place and I plan on making this much more customisable anyway so it should be a step in the right direction. And if it proves wrong in a later update we can always revert. :)